tst/EnergyPlus/unit/EMSManager.unit.cc

```diff
@@ -1114,12 +1114,11 @@ TEST_F(EnergyPlusFixture, EMSManager_TestFuntionCall)
                           ObjexxFCL::Optional_int_const()); // process trend functions again using above data
     EXPECT_TRUE(anyRan);
 
-    int index(0);
-    int offset(27); // first 26 values in ErlExpression() are key words + 1 EMS global variable
+    int offset = 28; // first 27 values in ErlExpression() are constant and built-in variables
```

Comment:
Fix existing test since there is now one more built-in var

Reply:
:+1: